sample/F8/meta

```diff
@@ -0,0 +1,3 @@
+{
+    "root_url": "http://open-event-dev.herokuapp.com/"
```

Comment:
`root_url` is url of server from where event is created.
please set it to `""` because it is not created using server